elasticsearch-monitoring/README.md

```diff
@@ -0,0 +1,68 @@
+# es-monitoring
+
+A Helm chart for monitoring Elasticsearch via Prometheus and Grafana. This chart will deploys the [elasticsearch-exporter](https://github.com/justwatchcom/elasticsearch_exporter) and configures Prometheus (via the [Operator](https://github.com/coreos/prometheus-operator)) for scraping the exposed metrics, sets up some alerts and Grafana dashboards (todo).
+
+## TL;DR
+
+```shell
+helm install skyscrapers/elasticsearch-monitoring -f values.yaml
+```
+
+## Prerequisites
+
+- Kubernetes 1.6+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```shell
+helm install --name my-release skyscrapers/elasticsearch-monitoring
+```
+
+The command deploys the complete Elasticsearch monitoring setup on the cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```shell
+helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
```

Comment:
Maybe also add a section about the limitations that you need to deploy it in the same namespace as your prometheus.

Reply:
Done.